src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map((style) => style.split(':'))
```

Comment:
I see u are using the `reduce` method. Why didn't u do it in the middle of the `reduce` method, on each iteration? + you can use convenient destructuring for this.
this is redundant
remove `map`